source/module_pw/pw_basis_big.h

```diff
@@ -137,6 +137,36 @@ class PW_Basis_Big: public PW_Basis
     lat.z = latvec.e33;
     ibox[2] = 2 * int(sqrt(gridecut_lat) * sqrt(lat * lat)) + 1;
 
+    int n1,n2,n3; 
```

Comment:
Please add some annotations to explain what these new code do.

Reply:
It solved #613 and pw_basis_init.cpp has added these code. However, I think pw_basis_big is a temperary class and will be removed fast. So I did not add these to pw_basis_big before.